platform/centec-arm64/linux-kernel-arm64.mk

```diff
@@ -0,0 +1,6 @@
+# linux kernel package for centec arm64
+
+LINUX_KERNEL_DTB_INITRAMFS = linux-dtb-initramfs.deb
+$(LINUX_KERNEL_DTB_INITRAMFS)_URL = https://github.com/CentecNetworks/sonic-binaries/raw/master/$(PLATFORM_ARCH)/kernel/$(LINUX_KERNEL_DTB_INITRAMFS)
```

Comment:
why download binary?

Reply:
linux-dtb-initramfs.deb contains the device tree and initrd.
The device tree is more fixed and is not submitted to the kernel as a patch.

These implementations reference marvell's:
sonic-buildimage/platform/marvell-arm64/linux-kernel-arm64.mk

Reply:
is this open sourced?

Reply:
This deb installer, which includes the device tree as dtb, also appends the kernel modules as ko rather than kernel patches, they are not open source.